WIP - {kokoro} Getting Snapshot tests working with bazel

romoore opened the pull request:
WIP - This needs to be broken-up into several PRs.

Commit:
Revert "Revert "[BottomAppBar] Fix it/bottom app bar cut out (#5958)""

This reverts commit e30327f8a710e7c35d37f9e1b7089eef357b5148.

Commit:
{kokoro} Getting Snapshot tests working with bazel

romoore commented on WORKSPACE:
```diff
@@ -88,3 +88,8 @@ git_repository(
     tag = "v6.0.0",
 )
+
+local_repository(
```
Blocked on https://github.com/material-foundation/ios-snapshot-test-case/pull/1

material-automation commented:
bazel detected changes to the following targets:

```
//components/BottomAppBar:unit_tests
//components/BottomAppBar:unit_tests_IPAD_PRO_12_9_IN_9_3
//components/BottomAppBar:unit_tests_IPAD_PRO_12_9_IN_9_3_test_bundle
//components/BottomAppBar:unit_tests_IPHONE_5_IN_8_1
//components/BottomAppBar:unit_tests_IPHONE_5_IN_8_1_test_bundle
//components/BottomAppBar:unit_tests_IPHONE_7_PLUS_IN_10_3
//components/BottomAppBar:unit_tests_IPHONE_7_PLUS_IN_10_3_test_bundle
//components/BottomAppBar:unit_tests_IPHONE_X_IN_11_0
//components/BottomAppBar:unit_tests_IPHONE_X_IN_11_0_test_bundle
//components/Chips:unit_tests
//components/Chips:unit_tests_IPAD_PRO_12_9_IN_9_3
//components/Chips:unit_tests_IPAD_PRO_12_9_IN_9_3_test_bundle
//components/Chips:unit_tests_IPHONE_5_IN_8_1
//components/Chips:unit_tests_IPHONE_5_IN_8_1_test_bundle
//components/Chips:unit_tests_IPHONE_7_PLUS_IN_10_3
//components/Chips:unit_tests_IPHONE_7_PLUS_IN_10_3_test_bundle
//components/Chips:unit_tests_IPHONE_X_IN_11_0
//components/Chips:unit_tests_IPHONE_X_IN_11_0_test_bundle
//components/TextFields:snapshot_tests
//components/TextFields:snapshot_tests_test_bundle
//components/TextFields:unit_tests
//components/TextFields:unit_tests_IPAD_PRO_12_9_IN_9_3
//components/TextFields:unit_tests_IPAD_PRO_12_9_IN_9_3_test_bundle
//components/TextFields:unit_tests_IPHONE_5_IN_8_1
//components/TextFields:unit_tests_IPHONE_5_IN_8_1_test_bundle
//components/TextFields:unit_tests_IPHONE_7_PLUS_IN_10_3
//components/TextFields:unit_tests_IPHONE_7_PLUS_IN_10_3_test_bundle
//components/TextFields:unit_tests_IPHONE_X_IN_11_0
//components/TextFields:unit_tests_IPHONE_X_IN_11_0_test_bundle
```

<!-- identifier: affected-targets -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // limitations under the License.
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerSnapshotTests.m -->

material-automation replied:
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountArabicSnapshotTests.m:
```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountArabicSnapshotTests.m:
```diff
@@ -12,10 +12,10 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // limitations under the License.
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountSnapshotTests.m -->

material-automation replied:
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerCharacterCountSnapshotTests.m:
```diff
@@ -12,7 +12,7 @@
 // limitations under the License.
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerCharacterCountSnapshotTests.m -->

material-automation replied:
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerCharacterCountSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCTextFieldSnapshotTestsStrings.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageSnapshotTests.m:
```diff
@@ -14,9 +14,9 @@
 
 #import <UIKit/UIKit.h>
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageSnapshotTests.m:
```diff
@@ -14,9 +14,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCTextFieldSnapshotTestsStrings.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountCyrillicSnapshotTests.m:
```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountCyrillicSnapshotTests.m:
```diff
@@ -12,10 +12,10 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerBaselineSnapshotTests.m:
```diff
@@ -12,7 +12,7 @@
 // limitations under the License.
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerBaselineSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerBaselineSnapshotTests.m:
```diff
@@ -12,7 +12,7 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerBaselineSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
-#import "MDCTextFieldSnapshotTestsStrings.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCTextFieldSnapshotTestsStrings.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineSnapshotTests.m:
```diff
@@ -12,7 +12,7 @@
 // limitations under the License.
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineSnapshotTests.m:
```diff
@@ -12,7 +12,7 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
-#import "MDCTextFieldSnapshotTestsStrings.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCTextFieldSnapshotTestsStrings.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCTextFieldSnapshotTestsStrings.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageSnapshotTests.m -->

Commit:
Snapshot renders are empty.

My suspicion is that the tests require a test_host. Working on a minimal
test host. Writing the rule is proving challenge (a dependency cycle
exists). It may be simpler to step back and get it working with a
per-component test host first, then migrate to a generalized rule.

material-automation commented on components/private/Snapshot/TestHost/src/main.m:
```diff
@@ -0,0 +1,22 @@
+  @autoreleasepool {
+    return UIApplicationMain(
+        argc, argv, nil,
+        NSStringFromClass([TestHostMinimalDelegate class]));
```
clang-format suggested the following change:


```
-    return UIApplicationMain(
-        argc, argv, nil,
-        NSStringFromClass([TestHostMinimalDelegate class]));
+    return UIApplicationMain(argc, argv, nil, NSStringFromClass([TestHostMinimalDelegate class]));
```

<!-- identifier: clang-format -->
<!-- File: components/private/Snapshot/TestHost/src/main.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerSnapshotTests.m:
```diff
@@ -12,7 +12,7 @@
 // limitations under the License.
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerSnapshotTests.m -->

material-automation replied:
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountArabicSnapshotTests.m:
```diff
@@ -12,12 +12,12 @@
-#import "MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountArabicSnapshotTests.m:
```diff
@@ -12,12 +12,12 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountKoreanSnapshotTests.m:
```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountKoreanSnapshotTests.m:
```diff
@@ -12,10 +12,10 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
-#import "MDCTextFieldSnapshotTestsStrings.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCTextFieldSnapshotTestsStrings.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
-#import "MDCTextFieldSnapshotTestsStrings.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCTextFieldSnapshotTestsStrings.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountKoreanSnapshotTests.m:
```diff
@@ -12,12 +12,12 @@
-#import "MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountKoreanSnapshotTests.m:
```diff
@@ -12,12 +12,12 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountCyrillicSnapshotTests.m:
```diff
@@ -12,12 +12,12 @@
-#import "MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountCyrillicSnapshotTests.m:
```diff
@@ -12,12 +12,12 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountHindiSnapshotTests.m:
```diff
@@ -12,12 +12,12 @@
-#import "MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountHindiSnapshotTests.m:
```diff
@@ -12,12 +12,12 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageArabicSnapshotTests.m -->

Commit:
Got them working and passing (without recording)

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFilledFloatingControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthCharacterCountSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthCharacterCountSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthCharacterCountSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
-#import "MDCTextFieldSnapshotTestsStrings.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCTextFieldSnapshotTestsStrings.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthCharacterCountSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/private/Snapshot/TestHost/src/TestHostMinimalDelegate.h:
```diff
@@ -0,0 +1,18 @@
+#import <UIKit/UIKit.h>
+
+// No-op delegate sufficient to satisfy UIApplicationMain.
+@interface TestHostMinimalDelegate : UIResponder<UIApplicationDelegate>
```
clang-format suggested the following change:


```
-@interface TestHostMinimalDelegate : UIResponder<UIApplicationDelegate>
+@interface TestHostMinimalDelegate : UIResponder <UIApplicationDelegate>
```

<!-- identifier: clang-format -->
<!-- File: components/private/Snapshot/TestHost/src/TestHostMinimalDelegate.h -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineCharacterCountSnapshotTests.m:
```diff
@@ -14,7 +14,7 @@
 #import <XCTest/XCTest.h>
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineCharacterCountSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineCharacterCountSnapshotTests.m:
```diff
@@ -14,7 +14,7 @@
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineCharacterCountSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerSnapshotTests.m:
```diff
@@ -12,7 +12,7 @@
 // limitations under the License.
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerSnapshotTests.m -->

material-automation replied:
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageCyrillicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageCyrillicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountSnapshotTests.m:
```diff
@@ -12,11 +12,11 @@
 // limitations under the License.
 
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountSnapshotTests.m:
```diff
@@ -12,11 +12,11 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
-#import "MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageSnapshotTests.m:
```diff
@@ -12,8 +12,8 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedFloatingControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```
clang-format suggested the following change:


```
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
-#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageHindiSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields+ColorThemer.h"
 #import "MaterialTextFields+TypographyThemer.h"
```
clang-format suggested the following change:


```
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldOutlinedControllerBaselineLeadingImageHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountHindiSnapshotTests.m:
```diff
@@ -12,10 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountHindiSnapshotTests.m:
```diff
@@ -12,10 +12,10 @@
-#import "MDCAbstractTextFieldSnapshotTests.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextAreaUnderlinedControllerCharacterCountHindiSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageKoreanSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldUnderlinedControllerLeadingImageKoreanSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```
clang-format suggested the following change:


```
+#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageArabicSnapshotTests.m -->

material-automation commented on components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageArabicSnapshotTests.m:
```diff
@@ -12,9 +12,9 @@
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+I18N.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests+LeadingImage.h"
+#import "supplemental/MDCAbstractTextFieldSnapshotTests.h"
 #import "MaterialTextFields.h"
```
clang-format suggested the following change:


```
-#import "MaterialTextFields.h"
```

<!-- identifier: clang-format -->
<!-- File: components/TextFields/tests/snapshot/MDCTextFieldFullWidthControllerLeadingImageArabicSnapshotTests.m -->

material-automation commented:
clang-format recommended changes.

Consider installing [git-clang-format](https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/git-clang-format) and running the following command:

```bash
git clang-format $(git merge-base origin/develop HEAD)
```

<!-- identifier: clang-format -->

Commit:
Refactoring test host app

Commit:
Merge branch 'develop' of github.com:material-components/material-components-ios into develop

Commit:
Merge branch 'develop' into bazel-snapshot

Commit:
Merge branch 'develop' into bazel-snapshot

Commit:
Merge branch 'develop' into bazel-snapshot

material-automation commented:
The following files are not in any BUILD targets:

```
components/BottomAppBar/examples/BottomAppBarTypicalUseExample.swift
```


<!-- identifier: bazel-rule-coverage -->

material-automation commented:
The following files are not in any BUILD targets:

```
components/BottomAppBar/examples/BottomAppBarTypicalUseExample.swift
```


<!-- identifier: bazel-rule-coverage -->

material-automation commented:
The following files are not in any BUILD targets:

```
components/BottomAppBar/examples/BottomAppBarTypicalUseExample.swift
```


<!-- identifier: bazel-rule-coverage -->

romoore commented on components/private/Snapshot/BUILD:
```diff
@@ -0,0 +1,29 @@
+mdc_public_objc_library(
+    name = "Snapshot",
+    deps = [
+        "@ios_snapshot_test_case//:SnapshotTestCase",
```
Blocked on https://github.com/material-foundation/ios-snapshot-test-case/pull/1

romoore replied:
Done

romoore commented on components/TextFields/BUILD:
```diff
@@ -94,6 +96,37 @@ mdc_objc_library(
+        ":FontThemer",
+        ":TextFields",
+        ":TypographyThemer",
+        "//components/private/Snapshot",
```
Blocked on https://github.com/material-foundation/ios-snapshot-test-case/pull/1

romoore replied:
Done

Commit:
Using remote for ios-snapshot-test-case

material-automation commented:
The following files are not in any BUILD targets:

```
components/BottomAppBar/examples/BottomAppBarTypicalUseExample.swift
```


<!-- identifier: bazel-rule-coverage -->

Commit:
Add missing CoreGraphics dependency

Commit:
Reverting apple_rules upgrade

material-automation commented:
The following files are not in any BUILD targets:

```
components/BottomAppBar/examples/BottomAppBarTypicalUseExample.swift
```


<!-- identifier: bazel-rule-coverage -->

material-automation commented:
The following files are not in any BUILD targets:

```
components/BottomAppBar/examples/BottomAppBarTypicalUseExample.swift
```


<!-- identifier: bazel-rule-coverage -->

Commit:
Adding snapshot test library rule

material-automation commented:
The following files are not in any BUILD targets:

```
components/BottomAppBar/examples/BottomAppBarTypicalUseExample.swift
```


<!-- identifier: bazel-rule-coverage -->

Commit:
Merge branch 'develop' into bazel-snapshot

material-automation commented:
The following files are not in any BUILD targets:

```
components/BottomAppBar/examples/BottomAppBarTypicalUseExample.swift
```


<!-- identifier: bazel-rule-coverage -->

Commit:
Merge branch 'develop' into bazel-snapshot

material-automation commented:
The following files are not in any BUILD targets:

```
components/BottomAppBar/examples/BottomAppBarTypicalUseExample.swift
```


<!-- identifier: bazel-rule-coverage -->
